pysqldb3/geopackage.py

```diff
@@ -273,11 +264,11 @@ def gpkg_to_shp(self, print_cmd=False):
             except subprocess.CalledProcessError as e:
```

Comment:
i assume this is due to different naming restrictions in geopackage vs shp, why not just sanitize the table names on output (or maybe gdal will do it for you?) otherwise wont this potentially skip some tables?  I think that could be problematic, so i would rather it fail if there is a name issue if renaming on export isnt an option, and maybe give you the table names that were the problem, then add a method to rename geopackage tables to this class so the user can fix it

Reply:
I have changed the logic so that it cleans the name if possible, or returns an error that the user must clean the gpkg table names before trying to load the package